extensions.json

```
@@ -1199,6 +1199,11 @@
       "id": "vortizhe.simple-ruby-erb",
       "repository": "https://github.com/vortizhe/vscode-ruby-erb"
     },
+    {
+      "id": "vsciot-vscode.vscode-arduino",
+      "repository": "https://github.com/microsoft/vscode-arduino",
+      "version": "0.3.4"
```

> Entry was created using `node add-extension https://github.com/microsoft/vscode-arduino`.

Hey, awesome that you've found the helper script!

Did you know that it also supports direct `.vsix` URLs via a `--download` flag?

I see that https://github.com/microsoft/vscode-arduino/releases also has an official `.vsix` release, so whenever that's the case, we usually prefer to re-publish that instead of attempting to repackage ourselves.

You can achieve this with this command:

```
node add-extension --download=https://github.com/microsoft/vscode-arduino/releases/download/v0.3.4/vscode-arduino-0.3.4.vsix
```

```suggestion
      "download": "https://github.com/microsoft/vscode-arduino/releases/download/v0.3.4/vscode-arduino-0.3.4.vsix",
      "version": "0.3.4"
```